lib/SQLiteReporter.cpp

```diff
@@ -215,6 +206,24 @@ void Mutang::SQLiteReporter::reportResults(const std::unique_ptr<Result> &result
   outs() << "Results can be found at '" << databasePath << "'\n";
 }
 
+#pragma mark -
+
+std::string SQLiteReporter::getCallerPathAsString(std::vector<std::string> callerPath) {
```

Comment:
Good idea to extract this in a separate function! Note though that you're passing the vector by value here, which means that you're making a copy. Better pass it by const reference.